src/components/records/RecordsList.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import useResource from '../../hooks/useResource';
+
+interface Record {
```

Comment:
Unless this is an internal type, please put all types in `types.ts` file. this can be specific for a section of the app or global (in the root). There can be several `types.ts` files